### PR TITLE
link klant for productaanvraag

### PR DESCRIPTION
**Pull request by ivo-ritense**

### CODE REVIEW

File: klant/src/main/kotlin/com/ritense/klant/service/impl/BurgerService.kt [hunk 1]
```diff
@@ -17,8 +17,43 @@
...
+    private val openKlantClientProperties: OpenKlantClientProperties,
+    private val openKlantClient: OpenKlantClient
+) : BurgerService {
     override fun getBurger(bsn: String) = openKlantClient.getKlant(bsn)
+
+    override fun createBurger(bsn: String): Klant {
+        val klantRequest = KlantCreationRequest(
+            openKlantClientProperties.rsin,
+            generateKlantNummer(),
+            "http://www.invalid-url.com/", // TODO: retrieve websiteUrl
```
[the remark this answers is not in the document]
Comment (ivo-ritense) [reply klant/src/main/kotlin/com/ritense/klant/service/impl/BurgerService.kt:37]: There is some temporary data in here that's ok for this PR. We don't have any other website URL and its required in the API. Later on we will discuss what to do with this.

File: klant/src/main/kotlin/com/ritense/klant/service/impl/BurgerService.kt [hunk 2]
```diff
@@ -17,8 +17,43 @@
...
+class BurgerService(
+    private val openKlantClientProperties: OpenKlantClientProperties,
+    private val openKlantClient: OpenKlantClient
+) : BurgerService {
     override fun getBurger(bsn: String) = openKlantClient.getKlant(bsn)
+
+    override fun createBurger(bsn: String): Klant {
+        val klantRequest = KlantCreationRequest(
+            openKlantClientProperties.rsin,
+            generateKlantNummer(),
```
[the remark this answers is not in the document]
Comment (ivo-ritense) [reply klant/src/main/kotlin/com/ritense/klant/service/impl/BurgerService.kt:36]: Again, temporary data for now. Should be changed before this goes in production.

**Review by Rick-Ritense — APPROVED** [review]

File: klant/src/main/kotlin/com/ritense/klant/service/impl/BurgerService.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (marijnritense) [reply klant/src/main/kotlin/com/ritense/klant/service/impl/BurgerService.kt:37]: We have a lot of temporary things. How are we keeping track? Mainly through TODOs right?

[the remark this answers is not in the document]
Comment (ivo-ritense) [reply klant/src/main/kotlin/com/ritense/klant/service/impl/BurgerService.kt:37]: Good idea. I did ask Peter about this issue and he's going to figure out what we want to do long term. I'll ask him if he has a preferred way.

**Review by marijnritense — APPROVED** [review]

**Review by Rick-Ritense — APPROVED** [review]

**Pull request merged**